code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -50,7 +48,15 @@
 			/mob/living/simple_animal/hostile/blob,
 			/mob/living/simple_animal/ascendant_shadowling
 			)//exclusion list for things you don't want the reaction to create.
-		var/list/critters = typesof(/mob/living/simple_animal/hostile) - blocked // list of possible hostile mobs
+		var/list/meancritters = typesof(/mob/living/simple_animal/hostile) - blocked // list of possible hostile mobs
+		var/list/nicecritters = list(typesof(/mob/living/simple_animal/pet),

```

Comment:
Is this valid syntax? I would think this would create a list within the list of those types(i.e. nicecritters = list(list(pet, cat, dog, pug...), crab, mouse...)